go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/argoproj-labs/argocd-operator
 
-go 1.21
+go 1.21.0
+
+toolchain go1.21.9
```

Comment:
I think we can remove this.

Reply:
Not sure how to do it. `go mod tidy` is adding it again even if I remove it and set to just `go 1.21`.